src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -0,0 +1,654 @@
+    * Symbolic links will be kept, if possible
+
+9. Make your own filter rules in a named file. For syntax, see e.g.
+    https://www.tutorialspoint.com/unix_commands/rsync.htm
+"""
+
+
+# FILTER* are per file filters, used in the second rsync ommand in the shell file below.
+# DIRFILTER* are per folder, used if DIRTREE == 1 or 2 in the shell script below.
+
+FILTER1 = """
++ **
+"""
+
+FILTER2 = """
+- backup/**
+- users/**
+- attic/**
+- .git/**
+- *.git
+- *.svn
+- *~
+"""
+
+DIRFILTER2 = """
+- backup/
+- users/
+- attic/
+- .git/
++ */
+- *
+"""
+
+FILTER5_ADD = """
++ rms/output/**
++ share/results/**
++ share/templates/**
+"""
+
+FILTER3_ADD = """
+- ert/output/**
+- ert/storage/**
+- ert/output/storage/**
+- input/seismic/**
+- rms/model/*.log
+- rms/output/**
+- spotfire/**/*.csv
+- spotfire/**/*.dxp
+- share/results/**
+- share/templates/**
+"""
+
+DIRFILTER3 = """
+- backup/
+- users/
+- attic/
+- .git/
+- ert/output/
+- ert/storage/
+- ert/**/storage/
+- rms/output
+- rms/input/seismic/
+- share/results/
+- share/templates/
++ */
+- *
+"""
+
+DIRFILTER5 = """
+- backup/
+- users/
+- attic/
+- .git/
+- ert/output/
+- ert/storage/
+- ert/**/storage/
+- rms/input/seismic/
++ */
+- *
+"""
+
+DIRFILTERX = """
++ */
+- *
+"""
+
+FILTER5 = FILTER2 + FILTER3_ADD + FILTER5_ADD
+FILTER3 = FILTER2 + FILTER3_ADD
+
+FILTER6 = """
++ share/coviz/**
+- *
+"""
+
+
+SHELLSCRIPT = """\
+#!/usr/bin/sh
+
+# SETUP OPTIONS
+
+SRCDIR="$1"  # a relative path
+DESTDIR="$2"  # must be an absolute path!
+FILTERFILE="$3"
+THREADS=$4
+RSYNCARGS="$5"
+DIRTREE=$6  # if 1 first copy folder tree, if 2 do it afterwards with dirfilterfile
+DIRFILTERFILE="$7"
+
+PWD=$(pwd)
+
+start=`date +%s.%N`
+
+cd $SRCDIR
+
+echo " ** Target folder is $DESTDIR"
+echo " ** Filter file is $FILTERFILE"
+mkdir -p $DESTDIR
+
+echo " ** Sync folders and files!"
+
+if [ $DIRTREE -eq 1 ]; then
+    echo " ** Sync all folders first..."  # this is usually fast
+    rsync -a -f"+ */" -f"- *" . $DESTDIR
+fi
+
+echo " ** Sync files using multiple threads..."
+find -L . -type f | xargs -n1 -P$THREADS -I% \
+    rsync $RSYNCARGS -f"merge $FILTERFILE" % $DESTDIR
+
+if [ $DIRTREE -eq 2 ]; then
+    echo " ** Sync all folders (also empty) except some..."
+    rsync -a -f"merge $DIRFILTERFILE" . $DESTDIR
+fi
+
+end=`date +%s.%N`
+
+runtime=$( echo "$end - $start" | bc -l )
+
+echo " ** Compute runtime..."
+
+echo $runtime
+cd $PWD
+
+"""
+
+
+def get_parser() -> argparse.ArgumentParser:
+    """Setup parser."""
+
+    usetext = "fmu_copy_revision <commandline> OR interactive"
+    parser = argparse.ArgumentParser(
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter,
+        usage=usetext,
+    )
+
+    parser.add_argument("--dryrun", action="store_true", help="Run dry run for testing")
+    parser.add_argument(
+        "--all, -all, -a", action="store_true", dest="all", help="List all folders"
+    )
+    parser.add_argument(
+        "--verbosity, -v",
```

Comment:
(this kind of option is named "verbose" in most other tools in subscript)

Reply:
OK, I add --verbose as first (--verbosity will still work also, as well as -v)